### DISCUSSION TITLE
Nous A1 (from 2022) updating from 12 to 14.3.0 Blocked Loop

### DISCUSSION THREAD

**Discussion by mannebk79** [arendst/Tasmota#22508] [Support request]
Hi there,

I rund like 50 nous a1, mainly for measuring stuff, bought somewhere during the start of 2022, probably the last version of the Gosund that was flashable with tuya ota update and back then was branded nous a1, the one that is rated at 16A but usually dies at anything above 10A. And does not like if you switch air or water valves through magnetic grid voltage ac coils

Ive been runing them for a while on 11, then upgraded to 12 and didnt touch the thing for way over a year.

Just recently set up a new home assistant, as I had to many things to fix with my end of 22 HA version. 

Part of that was Tasmota Admin. And while implemting the new HA i had some unused plugs that totally forgott what they where, a first try to fix was an update to 14.3 well that didnt fix it, but i discovered that i had no down falls of the 14.3

So when I had em all migrated to new WiFi, IP Range and setup sucessfully in HA, i gave Tasmota admin the go to take care of my lot to batch upgrade to 14.3

most of them took it well, asside from two.

one just took 10 mins to finish....

and one is stuck in "Blocked Loop" when ever I try to flash anything else than minimal.

I tried OTA from my tasmota admin, OTA from tasmota repository, and i tried uploading the bin files through the web interface. If i feed it .gz, as expected it reports file to big. If i feed it minimal.bin it upgrades to minimal.bin, if I feed it any other bin file, Blocked Loop 55

any quick and dirty recommendations?

All of my plugs came from the same firmware, they are all setup the same, asside from this one maybe having an a bit long name, however I have one that goes by 4 more letters and that one had no problem with the update. So im a bit confused what went wrong or if mybe I have an other one just dead of these units, they sure die like flys. 2,5 years old, I already lost about 35% and only one of these was due to me overloading it with 2,4kw continus from a washing machine.

The plug is not worth it to spend more than a buch of minutes fixing it, Im into ordering an other batch of nous a1t on the next sale on amazon, just returned a big lot of nous a1 with tuya firmware that I had no chacne of converting, cause I accidentially ordered the a1 (not a1t for tasmota)

Here is a screenshot from console

![grafik](https://github.com/user-attachments/assets/0bb64d56-06b5-4219-bebe-5fafcce313d9)

Thank you kindly
Regards
Manny

**Comment by barbudor** [comment]
What do you mean by 'blocked loop'?

I don't have as many as you (around 12 Gosund and 4 Nous A1T)
But with increase of feature in Tasmota which requires more CPU, I'm not using anymore the standard 'tasmota' build but a custom based in 'lite+energy'

**Comment by mannebk79** [comment]
well, it wokrs fine on 43 units, one took ages for the update, and one shows this:

![grafik](https://github.com/user-attachments/assets/128df491-4455-4e1b-8a14-3ad9ccedaf7d)

I did em all the same way, they are setup all the same way. I have a set of console strings for making em all the same asside from the naming and IP...

43 bueno, one semi bueno, and ones a mess

about 31 months ago i started with 1 for tests, then right away ordered an other 17 sets of 4, a total of 68 at 8something per unit on a special offer, way better than sonof via ali with basically no warranty at all.

Inside the first 6 months i did send 10 back, got only a refund for 8 from amazon as the 3rd box was half empty... i was expecting like 50% or so, but nada...

currently im down to 45, 65% of what i started with, looks like im short one more, again.

**Reply by sfromis** [reply]
Try not running TasmoAdmin (or similar tools) while upgrading. That can become "too much" for some "weak" devices.

**Comment by mannebk79** [comment]
thanks for that hint. 

let me concluded to make sure i got that right: so we have a batch tool, but cant use it cause the el cheapo china stuff, but only sometimes, and for completly unforeseeable reasons, cant handle it, and therefore, the solution is to go back to do manually upgrade runs.... thats fun in a mostly populated /23 network... (i recently spet about a day just to upgrade my 108v2 netgear silent managed switches, after half a day i decided to spend a few grands on new hardware with centralized update management...)

While waiting for other stuff i fiddled a bit more, and i followed the advice here and found i can get the "light" firmware to install.... looks to me like a corrupt flash memory area. i also can get it to go back to minimal...

but when ever i try a full firmware, that works fine on all the sister devices, it gets way sluggish and then ends in the boot loop.

is there a way to trigger some sort of mem test and have It drop the corrupted part? so i know whats still available and working?

or rather just declare it dead?
